blueflood-http/src/main/java/com/rackspacecloud/blueflood/http/HttpResponder.java

```diff
@@ -34,6 +36,10 @@ public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpRespo
     }
 
     public static void respond(ChannelHandlerContext ctx, HttpRequest req, HttpResponse res) {
+
+        // set response headers
+        res.setHeader("Access-Control-Allow-Origin", Configuration.getInstance().getStringProperty(CoreConfig.CORS_ALLOWED_ORIGINS));
```

Comment:
Also, do we need to get this property on every request? Can we get the value of it in the constructor and reuse?